k8s/aat/common/family-public-law/fpl-case-service.yaml

```diff
@@ -27,52 +27,11 @@ spec:
       image: hmctspublic.azurecr.io/fpl/case-service:prod-f4f18a39
       ingressHost: fpl-case-service-aat.service.core-compute-aat.internal
       environment:
-        IDAM_API_URL: https://idam-api.aat.platform.hmcts.net
-        IDAM_S2S_AUTH_URL: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
-        CORE_CASE_DATA_API_URL: http://ccd-data-store-api-aat.service.core-compute-aat.internal
-        DOCUMENT_MANAGEMENT_URL: http://dm-store-aat.service.core-compute-aat.internal
-        CCD_UI_BASE_URL: https://www-ccd.aat.platform.hmcts.net
```

Comment:
in chart definition it is http, here is https

Reply:
Updating in FPL https://github.com/hmcts/fpl-ccd-configuration/pull/940